src/Api/Endpoints/Projects/Candidates.php

```diff
@@ -0,0 +1,127 @@
+<?php
```

Comment:
Might be a good idea to use strict types in new classes from the start, rather than having to add them later?